### PR TITLE
Add AppendCommandTest & made small bug fixes

### PR DESCRIPTION
**Pull request by DonnyS57**
Added test for append command and made minor bug fixes! Will work on remove command test soon

### CODE REVIEW

File: src/main/java/donnafin/logic/commands/RemoveCommand.java [hunk 1]
```diff
@@ -24,11 +24,8 @@
 public class RemoveCommand extends Command {
 
     public static final String COMMAND_WORD = "remove";
-    public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": removes a row from the attribute table. "
-            + "Parameters: INDEX (must be a positive integer) "
-            + "Example: remove 3";
-    private static final String MESSAGE_SUCCESS = "Successfully removed row.";
+    public static final String MESSAGE_USAGE = ""; // TODO
+    private static final String MESSAGE_SUCCESS = ""; // TODO
```
Comment (bharathcs) [inline src/main/java/donnafin/logic/commands/RemoveCommand.java:28]:
```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD
            + ": removes a row from the attribute table. "
            + "Parameters: INDEX (must be a positive integer) "
            + "Example: remove 3";
    private static final String MESSAGE_SUCCESS = "Successfully removed row.";
```

File: src/main/java/donnafin/model/person/Policy.java [hunk 2]
```diff
@@ -25,7 +25,7 @@
                     new AttributeTable.ColumnConfig("Policy Name", "name", 300),
                     new AttributeTable.ColumnConfig("Insurer", "insurer", 100),
                     new AttributeTable.ColumnConfig("Insured Value", "totalValueInsuredToString", 100),
-                    new AttributeTable.ColumnConfig("Premium (yearly)", "yearlyPremiumsToString", 100),
+                    new AttributeTable.ColumnConfig("Premiums (Yr.)", "yearlyPremiumsToString", 100),
```
Comment (bharathcs) [inline src/main/java/donnafin/model/person/Policy.java:28]:
```suggestion
                    new AttributeTable.ColumnConfig("Premium (yearly)", "yearlyPremiumsToString", 100),
```

**Comment by bharathcs** [comment]
When you merged with master I think you didn't keep the changes made (like the command usage in remove), so I reverted your revert of those changes lol (6766a42)

**Review by bharathcs — DISMISSED** [review]
Nice work Donny, I like how its been done! Excellent positive test cases. In light of the #206  I believe we should add negative test cases for throwing errors on duplicate assets / policies / liabilities. No worries though, negative test cases and handling #206 and #207 certainly should be a new PR.

File: src/test/java/donnafin/logic/commands/AppendCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        try {
+            testPolicy = new Policy("Golden Mile", "AIA",
+                    "$2000", "$100", "$200");
+            testLiability = new Liability("Mortgage debt", "Debt", "$20000",
+                    "23 year loan from DBS Bank.");
+            testAsset = new Asset("Good Class Bungalow", "Property", "$2000000",
+            "Paid in full. No debt.");
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(Money.MESSAGE_CONSTRAINTS);
+        }
```
Comment (bharathcs) [inline src/test/java/donnafin/logic/commands/AppendCommandTest.java:68]: Hey, why are we throwing an error in this test?

File: src/test/java/donnafin/logic/commands/RemoveCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        try {
+            testPolicy = new Policy("Golden Mile", "AIA",
+                    "$2000", "$100", "$200");
+            testLiability = new Liability("Mortgage debt", "Debt", "$20000",
+                    "23 year loan from DBS Bank.");
+            testAsset = new Asset("Good Class Bungalow", "Property", "$2000000",
+                    "Paid in full. No debt.");
+        } catch (IllegalArgumentException e) {
+            throw new IllegalArgumentException(Money.MESSAGE_CONSTRAINTS);
+        }
```
Comment (bharathcs) [inline src/test/java/donnafin/logic/commands/RemoveCommandTest.java:76]: Same here.

File: src/test/java/donnafin/logic/commands/AppendCommandTest.java (hunk 3 above)
Comment (bharathcs) [inline src/test/java/donnafin/logic/commands/AppendCommandTest.java:68]: Why are we catching and rethrowing the same exception here? It doesn't make sense in a test i feel.

File: src/test/java/donnafin/logic/commands/RemoveCommandTest.java (hunk 4 above)
Comment (bharathcs) [inline src/test/java/donnafin/logic/commands/RemoveCommandTest.java:76]: Same here.

**Review by bharathcs — CHANGES_REQUESTED** [review]

File: src/test/java/donnafin/logic/commands/AppendCommandTest.java (hunk 3 above)
Comment (DonnyS57) [reply src/test/java/donnafin/logic/commands/AppendCommandTest.java:68]: Hmmm I think last time when I didn't surround it in a try-catch block it threw an error but now it doesn't.... I can just remove the try-catch block overall since it will always use a valid input (in this case).

File: src/test/java/donnafin/logic/commands/RemoveCommandTest.java (hunk 4 above)
Comment (DonnyS57) [reply src/test/java/donnafin/logic/commands/RemoveCommandTest.java:76]: same as previous comment

**Review by bharathcs — APPROVED** [review]

**Review by bharathcs — APPROVED** [review]
